packages/blockchain-wallet-v4-frontend/src/modals/Btc/AddBtcWallet/template.js

```diff
@@ -38,7 +38,7 @@ const AddBtcWallet = ({
     <Modal size='large' position={position} total={total}>
       <Form onSubmit={handleSubmit}>
         <Wrapper>
-          <ModalHeader icon='arrow-up-circle' onClose={close}>
+          <ModalHeader icon='arrow-up-circle' onClose={() => close()}>
```

Comment:
Why do we need the anonymous function here?

Reply:
it didn't worked other way, I didn't spent decent time to figure it out why